### PR TITLE
Add content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translate content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

### CODE REVIEW

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: kubeadm alpha
+content_type: conceito
+weight: 90
+---
+
+{{< caution >}}
+O `kubeadm alpha` fornece uma prévia de um conjunto de recursos disponibilizados para coletar feedback da comunidade. Por favor, experimente e nos dê feedback!
```
Comment (DonatoHorn) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:8]: suggestion: could be "experimente e nos dê um feedback!"
or "experimente e nos dê seu feedback!"
what you mean

**Review by DonatoHorn — COMMENTED** [review]
Good job Erlison!

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md (hunk 1 above)
Comment (MrErlison) [reply content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:8]: I liked.

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+{{< /caution >}}
+
+Atualmente, não há comandos experimentais sob `kubeadm alpha`.
+
+## {{% heading "O que vem a seguir?" %}}
```
[the remark this answers is not in the document]
Comment (MrErlison) [reply content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:13]: I need to fix this in all other files. Thank you.

File: content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+{{< caution >}}
+O `kubeadm alpha` fornece uma prévia de um conjunto de recursos disponibilizados para coletar feedback da comunidade. Por favor, experimente e nos dê seu feedback!
+{{< /caution >}}
+
+Atualmente, não há comandos experimentais sob o `kubeadm alpha`.
+
+## {{% heading "whatsnext" %}}
+
+* [kubeadm init](/docs/reference/setup-tools/kubeadm/kubeadm-init) para inicializar um nó do plano de controle do Kubernetes
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md:15]: "Camada de gerenciamento" instead of "plano de controle".

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hello there, @MrErlison! Thanks for your contribution! 🎉 

Very minor change required. Other than that it's a LGTM.

**Comment by rikatz** [comment]
/lgtm
/approve
Keep the contributions coming!! :D

**Pull request merged**